phpBB/language/en/common.php

```diff
@@ -190,24 +190,25 @@
 	'COOKIES_DELETED'		=> 'All board cookies successfully deleted.',
 	'CURRENT_TIME'			=> 'It is currently %s',
 
-	'DAY'					=> 'Day',
-	'DAYS'					=> 'Days',
-	'DELETE'				=> 'Delete',
-	'DELETE_ALL'			=> 'Delete all',
-	'DELETE_COOKIES'		=> 'Delete cookies',
-	'DELETE_MARKED'			=> 'Delete marked',
-	'DELETE_POST'			=> 'Delete post',
-	'DELIMITER'				=> 'Delimiter',
-	'DESCENDING'			=> 'Descending',
-	'DISABLED'				=> 'Disabled',
-	'DISPLAY'				=> 'Display',
-	'DISPLAY_GUESTS'		=> 'Display guests',
-	'DISPLAY_MESSAGES'		=> 'Display messages from previous',
-	'DISPLAY_POSTS'			=> 'Display posts from previous',
-	'DISPLAY_TOPICS'		=> 'Display topics from previous',
-	'DOWNLOADED'			=> 'Downloaded',
-	'DOWNLOADING_FILE'		=> 'Downloading file',
-	'DOWNLOAD_COUNTS'		=> array(
+	'DAY'						=> 'Day',
+	'DAYS'						=> 'Days',
+	'DELETE'					=> 'Delete',
+	'DELETE_ALL'				=> 'Delete all',
+	'DELETE_COOKIES'			=> 'Delete cookies',
+	'DELETE_MARKED'				=> 'Delete marked',
+	'DELETE_POST'				=> 'Delete post',
+	'DELIMITER'					=> 'Delimiter',
+	'DESCENDING'				=> 'Descending',
+	'DISABLED'					=> 'Disabled',
+	'DISPLAY'					=> 'Display',
+	'DISPLAY_GUESTS'			=> 'Display guests',
+	'DISPLAY_MESSAGES'			=> 'Display messages from previous',
+	'DISPLAY_POSTS'				=> 'Display posts from previous',
+	'DISPLAY_TOPICS'			=> 'Display topics from previous',
+	'DOMAIN_NO_MX_RECORD_EMAIL'	=> 'The entered email domain has no valid MX record.',
```

Comment:
This line was added here.
All other lines 'were edited', to have the proper indentation for the values (`=> '...',`)
Not sure if that will mess up the language differences between versions? Otherwise I'll unindent.